app/models/submission.rb

```diff
@@ -344,6 +346,16 @@ def can_view_plagiarism_report(type, user, session)
     )
   end
 
+  def user_can_peer_review?(user)
```

Comment:
This is a helper method just for the permission above, correct?  Let's change the name of this to be user_can_peer_review_plagiarism or something similar.